### PR TITLE
fix errcheck issue

### PR DESCRIPTION
**Pull request by TheInvincibleRalph**
#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/28520

```release-note
NONE
```

### CODE REVIEW

**Review by hanzei — APPROVED** [review]
LGTM :+1:

File: server/.golangci.yml [hunk 1]
```diff
@@ -86,9 +86,6 @@ issues:
         channels/api4/plugin.go|\
         channels/api4/plugin_test.go|\
         channels/api4/post_test.go|\
-        channels/api4/preference_test.go|\
-        channels/api4/reaction_test.go|\
```
Comment (hanzei) [inline server/.golangci.yml:90]: @TheInvincibleRalph That is the change in question. I think you introduced it via [`31a7b57` (#28618)](https://github.com/mattermost/mattermost/pull/28618/commits/31a7b57836fe883478688721c03429de68dd40d6).

Comment (TheInvincibleRalph) [reply server/.golangci.yml:90]: Oh, alright. I'll make the correction now. Thank you!

**Comment by TheInvincibleRalph** [comment]
Reverted the unrelated changes. Thank you, @jespino and @hanzei!

**Review by hanzei — APPROVED** [review]

**Comment by hanzei** [comment]
Thanks @TheInvincibleRalph :rocket:

**Pull request merged**